_snippets/graphql/add-members-to-tier.mdx

```diff
@@ -37,7 +37,7 @@ mutation addMembersToTier($input: AddMembersToTierInput!) {
         "companyId": "co_123"
       }
     ],
-    "tierId": "tier_123"
+    "tierIdentifier": { "tierId": "tier_123" }
```

Comment:
Can you switch all of these to use externalId ? That is the most useful example and what the updateTenantTier and updateCompanTier mutations do.